README.md

```diff
@@ -513,6 +519,8 @@ From your AWS Management Console, perform the following steps:
             "macie2:GetMacieSession",
             "macie2:GetFindings",
             "macie2:ListFindings",
+            "network-firewall:Describe*",
```

Comment:
Should these be the same as the ones above?

Reply:
I'm not sure as this is not the "specific" list but usually uses a more general one

```
This policy may be used to provide only exactly the specific permissions
currently used by JupiterOne. Using this policy will most certainly require you
to update the policy in the future as more APIs are called by JupiterOne.
```

There is a different section called Specific Permissions Policy

Reply:
Ah ok, that makes sense.